### PR TITLE
First draft of README for Dryad deposition

### PR DESCRIPTION
**Pull request by dlebauer**

### CODE REVIEW

**Comment by jeanetteclark** [comment]
This is looking good - nice integration of accessing the betydb to programmatically fill in the metadata fields.

As I mentioned in slack, I'm not so sure about including the genotypes individually. It might be a little outside the normal usage of the taxonomic field, but including them in the EML document would make them searchable on certain platforms.

Since this is for a data paper, it might be nice to document the data records as `dataTable`s in the dataset. Within the `physical` element there is a `distribution` element that where you can put pointers to the individual files at whatever location they live.

As a broader question -the degree to which Dryad can support EML documents? There is already much more information in here than what the DataCite schema supports (which I think is what Dryad uses)

File: README.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # Repository for the TERRA REF Data Paper
 
-
 This is a data descriptor, intended for submission to Nature Scientific Data as citable reference that briefly describes and points to our data products and computing pipeline.
 
 We have an inclusive opt-in authorship policy- if you contributed and your name isn't listed in the [author list](https://docs.google.com/spreadsheets/d/1FnaeJZ1A6r1fa3UvhfczMLKh5CWaSMErBRw4vTKgPe8/edit?usp=sharing) please read the [TERRA REF authorship guidelines](https://terraref.gitbooks.io/terraref-documentation/content/manuscripts-and-authorship.html) and [fill out the authorship form](https://goo.gl/forms/xgzvF2LppMAZrrHx2)
```
Comment (MagicMilly) [inline README.md:5]:
```suggestion
We have an inclusive opt-in authorship policy- if you contributed and your name isn't listed in the [author list](https://docs.google.com/spreadsheets/d/1FnaeJZ1A6r1fa3UvhfczMLKh5CWaSMErBRw4vTKgPe8/edit?usp=sharing) please read the [TERRA REF authorship guidelines](https://docs.terraref.org/user-manual/manuscripts-and-authorship-guidelines#co-authorship) and [fill out the authorship form](https://goo.gl/forms/xgzvF2LppMAZrrHx2)
```
For some reason, the original link took me to main terra ref docs page, but the link I suggested takes me to the specific authorship page.

**Comment by dlebauer** [comment]
The first draft of the readme is provided as a pdf in a pre-release of the github.com/terraref/data-publication repository. Closing this PR and will re-open a new PR when I get a draft of the data paper.

**Comment by jeanetteclark** [comment]
Hey David, I wanted to have a look at this today but can't find the PDF. Is it ready for review?

**Comment by dlebauer** [comment]
@jeanetteclark Thanks! I sent a link in Slack last week. Also attached to this pre-release https://github.com/terraref/data-publication/releases/tag/v0.1. Let me know if you have any questions!

**Comment by jeanetteclark** [comment]
Not sure if you are ready for full copy editing yet, but this looks really good. It certainly goes above and beyond a lot of the data documentation that I see. There are a few places with sentence fragments or slightly wonky formatting, but it looks like it is getting pretty close.

**Comment by dlebauer** [comment]
Moved dryad content to separate repository terraref/data-publication

**Pull request closed without merging**